### PR TITLE
Add Whisper model in CI

### PR DESCRIPTION
**Pull request by mszhanyi**
### Description
 Add Whisper Conversion and E2E into Big Models pipeline

### CODE REVIEW

**Review by snnn — APPROVED** [review]

File: onnxruntime/python/tools/transformers/models/whisper/benchmark.py [hunk 1]
```diff
@@ -411,6 +411,8 @@ def handle_output(output):
         logger.info(f"Generated token length: {len(actual_output)} tokens")
         transcription = args.processor.batch_decode(ort_outputs[0], skip_special_tokens=True)[0]
         logger.info(f"Transcription: {transcription}")
+        # print to stdout for comparsion
```
Comment (kunal-vaishnavi) [inline onnxruntime/python/tools/transformers/models/whisper/benchmark.py:414]:
```suggestion
        # print to stdout for comparison
```

File: onnxruntime/python/tools/transformers/models/whisper/benchmark.py [hunk 2]
```diff
@@ -411,6 +411,8 @@ def handle_output(output):
         logger.info(f"Generated token length: {len(actual_output)} tokens")
         transcription = args.processor.batch_decode(ort_outputs[0], skip_special_tokens=True)[0]
         logger.info(f"Transcription: {transcription}")
```
Comment (kunal-vaishnavi) [inline onnxruntime/python/tools/transformers/models/whisper/benchmark.py:413]: If you need to print to stdout for comparison, is this line needed then? We should not print the transcription twice.

**Comment by kunal-vaishnavi** [comment]
There are existing conversion and E2E tests in the below class. Can these be run in the CIs as well?

https://github.com/microsoft/onnxruntime/blob/6b73ab3e3e72a9f2008e8d0e221b0be77d2993b1/onnxruntime/test/python/transformers/test_generation.py#L287-L401

**Comment by kunal-vaishnavi** [comment]
We can pull the audio file from ONNX Runtime Extensions in the YML file instead of uploading it into the `whisper` folder. The audio file is already used for the Whisper tests in ONNX Runtime Extensions and it is available [here](https://github.com/microsoft/onnxruntime-extensions/blob/main/test/data/1272-141231-0002.mp3).

https://github.com/microsoft/onnxruntime/blob/87dce362a136e4a837a6bbfb23b0340b210b53ec/tools/ci_build/github/azure-pipelines/bigmodels-ci-pipeline.yml#L401-L422

For the transcription comparison, we can save the expected output in the machine and modify the above YML file as follows.

```
...
echo " the cut on his chest still dripping blood the ache of his overstrained eyes even the soaring arena around him with the thousands of spectators were trivialities not worth thinking about" > expected_output.txt
diff ort_output.txt expected_output.txt && exit 0 || exit 1
...
```

With these changes, the `test` folder and its contents can be removed because they are not needed in the `whisper` folder.

**Comment by mszhanyi** [comment]
> We can pull the audio file from ONNX Runtime Extensions in the YML file instead of uploading it into the `whisper` folder. The audio file is already used for the Whisper tests in ONNX Runtime Extensions and it is available [here](https://github.com/microsoft/onnxruntime-extensions/blob/main/test/data/1272-141231-0002.mp3).
> 
> https://github.com/microsoft/onnxruntime/blob/87dce362a136e4a837a6bbfb23b0340b210b53ec/tools/ci_build/github/azure-pipelines/bigmodels-ci-pipeline.yml#L401-L422
> 
> For the transcription comparison, we can save the expected output in the machine and modify the above YML file as follows.
> 
> ```
> ...
> echo " the cut on his chest still dripping blood the ache of his overstrained eyes even the soaring arena around him with the thousands of spectators were trivialities not worth thinking about" > expected_output.txt
> diff ort_output.txt expected_output.txt && exit 0 || exit 1
> ...
> ```
> 
> With these changes, the `test` folder and its contents can be removed because they are not needed in the `whisper` folder.

The purpose is to reduce downloads as possible to increase the stability of CI.

**Comment by mszhanyi** [comment]
> There are existing conversion and E2E tests in the below class. Can these be run in the CIs as well?
> 
> https://github.com/microsoft/onnxruntime/blob/6b73ab3e3e72a9f2008e8d0e221b0be77d2993b1/onnxruntime/test/python/transformers/test_generation.py#L287-L401

 I think we could add it to next PR.

**Comment by kunal-vaishnavi** [comment]
Can we move the Whisper test files to the [existing Whisper test directory](https://github.com/microsoft/onnxruntime/tree/main/onnxruntime/test/python/transformers/test_data/models/whisper) then?

**Pull request merged**